Driver for new device #431, Smoke Sensor

WJGvdVelden opened the pull request:
Driver for New device. Smoke Sensor Issue #431
Device Name: _TZE200_ntcy3xu1
Device Model: TS0601

WJGvdVelden committed:
Add files via upload

Driver for New device. Smoke Sensor Issue #431

WJGvdVelden committed:
_TZ3000_b28wrpvx added 

This device is not a smart plug but a wall-outlet. Works fine with the same driver.

JohanBendz commented:
Hi @WJGvdVelden, thanks for the PR.
I see you added a commit that added _TZ3000_b28wrpvx to the app, while doing so you removed and changed other codes for the same driver, why so? I rather not have changes made to other drivers in a PR for a new device as it's easy to break stuff. Can you please remove that code so that I can accept your PR for the smoke sensor.

WJGvdVelden committed:
Reverting change to driver

WJGvdVelden commented:
Hi
Sorry, I had no idea that driver was also affecting this particular PR.
I can't revert the change on the website and I don't use the desktop version, so I have replaced that file with your current one to undo any changes to it. I hope that does the trick.

Thanks for looking into it.

Regards

Wim

________________________________
Van: Johan Bendz ***@***.***>
Verzonden: donderdag 19 oktober 2023 22:23
Aan: JohanBendz/com.tuya.zigbee ***@***.***>
CC: WJGvdVelden ***@***.***>; Mention ***@***.***>
Onderwerp: Re: [JohanBendz/com.tuya.zigbee] Driver for new device #431, Smoke Sensor (PR #481)


Hi @WJGvdVelden<https://github.com/WJGvdVelden>, thanks for the PR.
I see you added a commit that added _TZ3000_b28wrpvx to the app, while doing so you removed and changed other codes for the same driver, why so? I rather not have changes made to other drivers in a PR for a new device as it's easy to break stuff. Can you please remove that code so that I can accept your PR for the smoke sensor.

—
Reply to this email directly, view it on GitHub<https://github.com/JohanBendz/com.tuya.zigbee/pull/481#issuecomment-1771653265>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AUXB5HEL33CQSXMSD7YAU4TYAGD5HAVCNFSM6AAAAAA6BAWC3GVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTONZRGY2TGMRWGU>.
You are receiving this because you were mentioned.

WJGvdVelden committed:
Revert "Reverting change to driver"

This reverts commit b32f3c2521a6ec670062eeac4d4e90f9987a0c5f.

WJGvdVelden committed:
Revert "_TZ3000_b28wrpvx added "

This reverts commit 9026d7b3844a3f59051edffad0ac1dfe15e5b576.

WJGvdVelden committed:
New icon

JohanBendz commented:
Thanks @WJGvdVelden, that works! :)

Accepting the PR and merging it.

Well done! :)
